pkgs/applications/science/math/sage/sagelib.nix

```diff
@@ -119,6 +120,7 @@ buildPythonPackage rec {
     givaro
     glpk
     gsl
+    libpng
```

Comment:
Should this be in `propagatedBuildInputs`? Tests also seem to pass changing `buildInputs` instead.

Reply:
Let's give it a shot.